.github/workflows/test.yml

```diff
@@ -15,12 +15,10 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 10.x
-          - 11.x
-          - 12.x
-          - 13.x
-          - 14.x
-          - 15.x
```

Comment:
Are we removing support for versions 10 - 15?
If so can we update: https://github.com/Mastercard/client-encryption-nodejs?tab=readme-ov-file#compatibility- to keep everything consistent?

Reply:
Will do